NvimServer/NvimServer/bin/clean_all.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+readonly clean_deps=${clean_deps:-true}
+
+pushd "$(dirname "${BASH_SOURCE[0]}")/../.." >/dev/null
+  pushd ../Neovim
+
+  rm -rf ./build
+  rm -rf ./.deps
+  make distclean
```

Comment:
I'm getting `make: *** No rule to make target `distclean'` errors here. Do I miss something?

Reply:
May have forgotten to update the clean_all script. Or possibly stock vim only has make clean. I’ll look into that.

Reply:
@wookayin: Yes, Neovim has the `distclean` target. Since the NvimServer folder is almost empty in this PR, `make distclean` won't work. I'll have a call with @georgeharker in a few days to align on how to build Neovim. Afterwards, we'll tidy up the scripts.